bin/cheribsdtest/cheribsdtest_bounds_stack.c

```diff
@@ -70,14 +70,13 @@
 static void
 test_bounds_precise(void * __capability c, size_t expected_len)
 {
-	size_t len, offset;
-
-	offset = cheri_getoffset(c);
-	len = cheri_getlen(c);
-
 #ifdef __CHERI_PURE_CAPABILITY__
 	CHERIBSDTEST_CHECK_CAP_BOUNDS_PRECISE(c, expected_len);
 #else
+	size_t len;
```

Comment:
Moving this below the assertion seems a bit odd.

Reply:
I was trying to minimize #ifdef's, but it is a bit unusual, yes.

Reply:
Flipping the #ifdef around is a bit more aesthetically pleasing without having an #ifdef maze so I will go with that.